frontend/src/modules/LoginRedux.js

```diff
@@ -8,8 +10,22 @@ export const IS_CHANGING_PASSWORD = "IS_CHANGING_PASSWORD";
 export const CHANGE_PASSWORD_SUCCESS = "CHANGE_PASSWORD_SUCCESS";
 export const CHANGE_PASSWORD_FAILURE = "CHANGE_PASSWORD_FAILURE";
 
-// Simplified version of the authentication action (temporary)
-const authenticateAction = authenticated => ({ type: AUTHENTICATED, authenticated });
+function authenticateAction(response, dispatch, location, push) {
+  return async function() {
+    if (navigator.cookieEnabled) {
+      localStorage.setItem("token", response.token);
```

Comment:
why did you rename it from "auth_token"? that seemed more explicit.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/312/commits/793655aa429516a071f45af6a8d24061dd77adbe